agentops/__init__.py

```diff
@@ -180,6 +219,17 @@ def add_tags(tags: List[str]):
     Args:
         tags (List[str]): The list of tags to append.
     """
+    if Client().is_multi_session:
+        return logger.warning(
```

Comment:
def not right now, but we could create a logger mock and test all of these down the road